src/main.cpp

```diff
@@ -429,7 +429,6 @@ bool isEasterMode(int day, int month, int year, int before_after_days)
         }
         return (month > easter_start_month || (month == easter_start_month && day >= easter_start_day)) &&
                (month < easter_end_month   || (month == easter_end_month   && day <= easter_end_day));
-        break;
     }
```

Comment:
I would leave the break here. Having it does not affect anything negatively, but if that that code should be rewritten later (and the return removed), that could cause some unnecessary time to debug.